scripts/configure_nexus.sh

```diff
@@ -102,7 +102,7 @@ if [[ ${STATUS_CODE} == 404 ]]
 fi
 
 #Check if the repo already exists
-export STATUS_CODE=$(curl -iu admin:$NEXUS_PASS -X "GET" "${NEXUS_URL}/service/rest/v1/repositories/apt/proxy/pypi" -H "accept: application/json" -k -s -w "%{http_code}" -o /dev/null)
+export STATUS_CODE=$(curl -iu admin:$NEXUS_PASS -X "GET" "${NEXUS_URL}/service/rest/v1/repositories/proxy/pypi" -H "accept: application/json" -k -s -w "%{http_code}" -o /dev/null)
```

Comment:
I'm probably missing something, but why remove the `apt` from the URL?

Reply:
Did this as a error correct as there is both a pypi_proxy_conf.json and apt-pypi_proxy_conf.json.
One is an apt repo and one is not. The curl request checks to see if the repo already exists before creating it so the standard proxy will never be created at the moment due to this error.

Reply:
Ah, right. Got it.